.vscode/settings.json

```diff
@@ -8,6 +8,5 @@
 	"prettier.singleQuote": true,
 	"prettier.useTabs": true,
 	"prettier.trailingComma": "all",
-	"prettier.printWidth": 120,
-	"launchdarkly.sdkKey": ""
+	"prettier.printWidth": 120
```

Comment:
Can we just add this to gitignore as part of PR?

Reply:
Yeah... not sure why it's there. There's already a prettierrc. I'll remove it.